src/content/nft/index.md

```diff
@@ -108,7 +108,15 @@ NFT's have some special properties:
 In other words, if you _own_ an NFT:
 
 - You can easily prove you own it.
-  - Proving you own an NFT is very similar to proving you have ETH in your account
+  - Proving you own an NFT is very similar to proving you have ETH in your account.
+  - For example, lets say you pay for an NFT. What you are getting is the right to transfer your token to your digital wallet.
```

Comment:
"lets" -> "let's"

Consider:
"For example, let's say you pay for an NFT, and the ownership of this unique token is transferred to your wallet via your public address."